org.gnome.PowerStats.json

```diff
@@ -0,0 +1,68 @@
+{
+  "app-id": "org.gnome.PowerStats",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "command": "gnome-power-statistics",
+  "finish-args": [
+    "--share=ipc",
+    "--system-talk-name=org.freedesktop.UPower"
+  ],
+  "cleanup": [
+     "*.la",
+    "/include",
+    "/share/man"
+  ],
+  "modules": [
+    "shared-modules/udev/udev-175.json",
+    {
+      "name": "libusb",
```

Comment:
See also:
https://github.com/flathub/com.hamrick.VueScan/blob/46296c2d2c19c48ef9d9412e14d6d67c7e0e7b54/com.hamrick.VueScan.yaml#L21-L38

Reply:
I've copied the flags from there, I do wonder if libusb should be in shared-modules -- it's used by a lot of projects...